src/requests/features.jl

```diff
@@ -305,3 +305,22 @@ function _binding_kind(b ,server)
         return 13
     end
 end
+
+JSONRPC.parse_params(::Type{Val{Symbol("julia/getCurrentModule")}}, params) = TextDocumentPositionParams(params)
+function process(r::JSONRPC.Request{Val{Symbol("julia/getCurrentModule")},TextDocumentPositionParams}, server)
```

Comment:
There isn’t really a sense of `current` in this API, right? Maybe `getModuleAt` or `getSurroundingModule` or something like that?

Reply:
Makes sense - also are we happy to use "julia/..." for these arbitrary requests?